jmeter-java-dsl-csv-random/src/test/java/us/abstracta/jmeter/javadsl/csvrandom/RandomCsvDataTest.java

```diff
@@ -0,0 +1,38 @@
+package us.abstracta.jmeter.javadsl.csvrandom;
+
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static us.abstracta.jmeter.javadsl.JmeterDsl.testPlan;
+import static us.abstracta.jmeter.javadsl.JmeterDsl.threadGroup;
+
+import static us.abstracta.jmeter.javadsl.csvrandom.RandomCsvDataSetConfig.csvRandomDataSet;
+import static us.abstracta.jmeter.javadsl.JmeterDsl.jsr223Sampler;
+import org.junit.jupiter.api.Test;
+import us.abstracta.jmeter.javadsl.JmeterDslTest;
+import us.abstracta.jmeter.javadsl.core.TestPlanStats;
+
+import java.io.File;
+import java.net.URL;
+import java.nio.file.Paths;
+import java.time.Duration;
+
+public class RandomCsvDataTest extends JmeterDslTest {
```

Comment:
if you don't use wiremock or any of provided jmeterDslTest logic, maybe you don't need to extend it?